data/the emerald sword.txt

```diff
@@ -0,0 +1,803 @@
+			`	It's strange seeing so many technological devices as old as some of the oldest prehistorical tools on earth, most of them so deteriorated you wouldn't even be able to imagine what they were used for, if it wasn't for the descriptions in each of the displays. One of the most intact things is a small metal statue of a Sheragi in flight, possibly a trophy for some event, or merely a sculpture of a videogame character. It's impossible to know, but it reveals their physical form. As you look at how tall the doors of this house is, and back at the statue, you realize how big they must have been. It's hard to imagine the size a spaceship must be to hope to fit a creature this large. After the tour is over, he leads you to one of the buildings that are off-limits to the general public.`
+				goto offlimit
+			label hurry
+			`	"Straight to the point? Fair enough, I'm eager as well, but do come visit the museum some time in the future. There are so many interesting objects."`
+			`	You two catch up as he leads you into the ruins, talking about the "minor" discoveries he made so far, but also about an artifact found buried underground. He leads you into one of the buildings that are off-limits to the public.`
+			label offlimit
+			`	"Here it is!"`
+			`	There is a tunnel going deep underground in the room, but Albert has his attention on a safe. He elaborates as he opens it up`
+			`	"So, the Sheragi apparently died off in a war of catastrophic proportions. One of our interns suggested we take a closer look underground, to check for military facilities or bunkers, even though our previous scans didn't show anything interesting, so we left her to play with the ground penetrating radar. and wouldn't you believe, there were some angled pieces of metal under the earth. Clearly man made. Or well, Sheragi-made in this case. It was a bunker, that apparently got hit by a powerful kinetic weapon, but we could still salvage some artifacts from the debris. Particularly, this thing."`
+			`	Albert opens the safe, and shows you a weathered metal cube, around 40 cm on each side. he struggles with the weight, but manages to set it down on the floor.`
+			`	"This appears to be a data storage device of some sort. It was connected to what we think were once computers, but nothing else is salvageable after spending thousands of years under wet dirt. Based on how sturdy this thing is, it's plausible that some of the data could have survived, like a time capsule."`
+			`	"I have a contact back on vinci who could probably figure this out, and this is where you come in. You remember John, right? I can't leave the dig site, but I also don't have anyone else I could trust to carry such a valuable artifact. Will you take it there for me?"`
+			choice
+				`	"Sure, I'll take the cube to him."`
+					goto take
+				`	"I'm sorry, but that sounds sketchier than all the other jobs I did for you. The quarg were clear in saying that some things are not meant to be found."`
```

Comment:
It is because this is presumably something the Quarg wouldn't want humans to find. Getting in trouble with the Quarg would be an order of magnitude worse than getting in trouble with the Deep.